src/main/java/seedu/address/logic/parser/PriorityCommandParser.java

```diff
@@ -44,7 +43,7 @@ public PriorityCommand parse(String args) throws ParseException {
         String nricStr = argMultimap.getValue(PREFIX_NRIC).orElse("");
         Nric nric = ParserUtil.parseNric(nricStr);
         String priorityStr = argMultimap.getValue(PREFIX_PRIORITY).orElse("");
-        Priority priority = ParserUtil.parsePriority(priorityStr);
+        Priority priority = ParserUtil.parsePriority(priorityStr.toUpperCase());
```

Comment:
Can push the toUpperCase to inside `parsePriority` to maintain SLAP here and prevent possible repeating

Reply:
Amended